src/encoded/static/components/experiment_series.js

```diff
@@ -243,6 +496,15 @@ class ExperimentSeriesComponent extends React.Component {
                                     </div>
                                 : null}
 
+                                {context.target && context.target.length > 0 ?
+                                    <div data-test="target">
+                                        <dt>Target</dt>
+                                        <dd>
+                                            {context.target.map((targetObj, i) => <span key={i}>{i !== 0 ? ', ' : ''}<a href={targetObj['@id']}>{targetObj.label}{' ('}{targetObj.organism ? <i>{targetObj.organism.scientific_name}</i> : <span>{targetObj.investigated_as[0]}</span>}{')'}</a></span>)}
```

Comment:
Can you use `targetObj['@id']` as the `<span>` key instead of `i`? That way the key moves with the data as I mentioned.

Reply:
Yes